gputop-csv/gputop-csv.js

```diff
@@ -350,6 +406,7 @@ GputopCSV.prototype.update_features = function(features)
                             this.column_titles_.map((line) => {
                                 this.stream.write(line + this.endl);
                             });
+
```

Comment:
Please remove the newlines.

Reply:
Sorry for my mistake, I have removed this line.